py/pysparkling/ml/algo.py

```diff
@@ -0,0 +1,76 @@
+from pyspark import since, keyword_only
+from pyspark.ml.param.shared import *
+from pyspark.ml.util import JavaMLReadable, JavaMLWritable
+from pyspark.ml.wrapper import JavaEstimator, JavaModel, JavaTransformer, _jvm
+from pyspark.sql import SparkSession
+from pysparkling import *
+from .params import H2OAlgorithmParams
+
+
+class H2OGBM(JavaEstimator, H2OAlgorithmParams, JavaMLReadable, JavaMLWritable):
+
+    @keyword_only
+    def __init__(self, ratio=1.0, featuresCols=[], predictionCol=None):
+        super(H2OGBM, self).__init__()
+        self._java_obj = self._new_java_obj("org.apache.spark.ml.h2o.algos.H2OGBM",
+                                            self.uid,
+                                            H2OContext.getOrCreate(SparkSession.builder.getOrCreate(), verbose=False)._jhc.h2oContext(),
+                                            H2OContext.getOrCreate(SparkSession.builder.getOrCreate(), verbose=False)._jsql_context
+                                            )
+        self._setDefault( ratio=1.0, featuresCols=[], predictionCol=None)
+        kwargs = self._input_kwargs
+        self.setParams(**kwargs)
+
+    @keyword_only
+    def setParams(self, ratio=1.0, featuresCols=[], predictionCol=None):
+        kwargs = self._input_kwargs
+        return self._set(**kwargs)
+
+    def _create_model(self, java_model):
+        return H2OGBMModel(java_model)
+
+
+class H2OGBMModel(JavaModel, JavaMLWritable, JavaMLReadable):
+    pass
```

Comment:
How do i create an instance of `H2OGBMModel` if i have MOJO in hand?

Reply:
This I still need to figure out. But I think that we just need to create a helper method on python side similar to ```def createFromMojo(is: InputStream, uid: String = Identifiable.randomUID("mojoModel")): H2OMOJOModel = {```

which would call this method using py4j and then use this as JavaModel

This is jira for it, marked for the release after this one https://0xdata.atlassian.net/browse/SW-611

Reply:
Already resolved in the latest code